libsolidity/interface/Natspec.cpp

```diff
@@ -129,12 +166,6 @@ Json::Value Natspec::devDocumentation(std::multimap<std::string, DocTag> const&
 	if (!author.empty())
 		json["author"] = author;
 
-	// for constructors, the "return" node will never exist. invalid tags
```

Comment:
Do we have a test case for this?

Reply:
Does this one not provide proper coverage?

https://github.com/gh1dra/solidity/blob/29ec9445d859f425f97f54418912f29967a1a568/test/libsolidity/SolidityNatspecJSON.cpp#L831-L844

Reply:
I'll add one for the new error introduced as well.

Reply:
Yes, you're right. It's already partially covered. But a new one would be good.